### PR TITLE
Adjust columns on mobile view for step-list--detailed

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Sets columns to 6 at a smaller breakpoint

Fixes https://github.com/canonical/vanilla-framework/issues/5002

## QA

- Open [demo](https://vanilla-framework-5007.demos.haus/docs/examples/patterns/lists/lists-stepped-detailed)
- Check that there is the content and title of the stepped list take the appropriate number of columns
- On merge remove this from ubuntu.com https://github.com/canonical/ubuntu.com/pull/13580/files#diff-f5a5b0429dba7d86dd96396c39539ab3a091d90536bccb9db008a3dbc487f83dR1506

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [what's new page](https://github.com/canonical/vanilla-framework/blob/main/releases.yml).

## Screenshots

Mobile
image

Tablet
image

Desktop
image

### CODE REVIEW

File: releases.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- version: 4.8.0
+- version: 4.8.1
```
Comment (bartaz) [inline releases.yml:1]: If we are not adding anything to changelog we should not change the version (especially that it's the one already released).

So please revert this change.

```suggestion
- version: 4.8.0
```

**Comment by bartaz** [comment]
@petesfrench There is some prettier formatting issue in one of the files.

Running `dotrun test` locally should catch it as well. Or if you have prettier VS Code plugin, it should highlight it as well.

File: templates/docs/examples/base/playground.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Links{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
```
Comment (bartaz) [inline templates/docs/examples/base/playground.html:6]: Was this intended to be part of this PR?

Comment (petesfrench) [reply templates/docs/examples/base/playground.html]: Nope, removed

**Review by bartaz — APPROVED** [review]
